shentu-2.2/distribution/README.md

```diff
@@ -0,0 +1,9 @@
+
+
+### Estimated distribution time
+- September 27, 2023, 13:00 UTC
+
+### Distribution rewards through DeepWallet
```

Comment:
Should we use the wording "Loyalty Gratitude Distribution"?

Reply:
we need use this term together with the button